pkg/hsm/awscloudhsm/cloudhsm.go

```diff
@@ -157,10 +157,13 @@ func (c *Client) GenerateAndSignMetadata(request hsm.GenerateMetadataRequest) (s
 		tmpMetadataSigningCertPath,
 		"--output", tmpMetadataOutputPath,
 		"--algorithm", "rsa",
+		"--hsm-metadata-signing-label", request.MetadataSigningKeyLabel,
```

Comment:
Think you need to drop these lines. The params were changed to have "key-label" as the suffix. They're included just below.